test/data/200.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Can this file be made a bit smaller? We don't need all of the fields for this test.

Reply:
Done